src/routes.js

```diff
@@ -1,14 +1,16 @@
 /* Libraries */
 import React from 'react';
-import { Route } from 'react-router';
+import { Route, IndexRoute } from 'react-router';
```

Comment:
Just for organization can we reorder it so it's IndexRoute, Route (in alphabetical)